executor-service/src/main/java/executor/service/model/ThreadPoolConfig.java

```diff
@@ -1,10 +1,18 @@
 package executor.service.model;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
 import java.util.Objects;
 
+@Component
+@PropertySource("classpath:config.properties")
 public class ThreadPoolConfig {
```

Comment:
it may be better to put these classes in AppConfig and use `@Configuration` + `@Bean`
Read from one file `application.properties` and use `@ConfigurationProperties(prefix = "thread-pool")`

Reply:
done, thanks